__tests__/core/reflection.test.ts

```diff
@@ -184,6 +185,8 @@ test("reflection - members chained", () => {
   expect(reflection.actions.includes("anotherAction")).toBe(true)
   expect(reflection.views.includes("viewName")).toBe(true)
   expect(reflection.views.includes("anotherView")).toBe(true)
+  expect(reflection.views.includes("actionName")).toBe(false)
+  expect(reflection.views.includes("anotherAction")).toBe(false)
```

Comment:
**praise:** @evinosheaforward - thanks for updating the test!

**issue (blocking):** can we also update these tests so they exercise the inline docs listed in `src/core/mst-operations.ts:866`, which say:

>  In the case where a model has two actions: `doSomething` and `doSomethingWithFlow`, where `doSomethingWithFlow` is a flow generator, the `actions` array will contain both actions, i.e. ["doSomething", "doSomethingWithFlow"], and the `flowActions` array will contain only the flow action, i.e. ["doSomethingWithFlow"].

**issue (blocking):** there's also an opportunity for us to exercise that same functionality in the `reflection - model` test in `__tests__/core/reflection.test.ts:59`

Let me know if you want a clearer suggestion on what to do with those, but I'd like to take this moment to improve our test coverage around this feature while we're looking at it.

Reply:
Thanks for the feedback!
>issue (blocking): there's also an opportunity for us to exercise that same functionality in the reflection - model test in __tests__/core/reflection.test.ts:59

I added the explicit test that the flow action is added to `actions` in the `reflection - model` test, good call.

>issue (blocking): can we also update these tests

I'd be glad to update this test, but it doesn't currently have any flow actions.
If you'd like me to add one, let me know if there the example from the other test would work:
```
  .actions((self) => {
    function actionName() {
      return 1
    }
    return {
      actionName,
      generatorAction: flow(function* generatorAction() {
        const promise = new Promise((resolve) => {
          resolve(true)
        })
        yield promise
      })
    }
  })
```

Reply:
Yup, that'd be perfect, thank you!

Reply:
addressed this!

I also added a little line to confirm that an action associated with a flowaction is not added to the flowactions in getMembers. I saw no downside and potential upside, if you disagree just lmk!

Reply:
Nailed it! Will merge and get a patch release out sometime this week!